libr/core/cmd_anal.c

```diff
@@ -1899,7 +1918,7 @@ void cmd_anal_reg (RCore *core, const char *str) {
 	}
 }
 
-static int esil_step(RCore *core, ut64 until_addr, const char *until_expr) {
+R_API int esil_step(RCore *core, ut64 until_addr, const char *until_expr) {
```

Comment:
what do you mean by `if the function doesnt matches the signature prefix of the module`